app/build.gradle

```diff
@@ -5,12 +5,13 @@ plugins {
 apply from: '../dependencies.gradle'
 
 android {
-    compileSdk 32
+    namespace = "com.tomtom.sdk.examples"
+    compileSdk 33
 
     defaultConfig {
         applicationId "com.tomtom.sdk.examples"
         minSdk 24
```

Comment:
Not related to this PR, but I've noticed that according to our current Project Setup guide, the minimum SDK API level should be 21. Do we have an inconsistency here? Do we need to update the guide?
@AlinaUshakova-TomTom

Reply:
It is that it should be at least 21, below that the SDK will not work